stories/MewCheckbox/MewCheckbox.stories.js

```diff
@@ -30,6 +30,9 @@ export const MEWCheckbox = () => ({
     },
     link: {
       default: object('link', linkObj)
+    },
+    isChecked: {
+      default: boolean('checked?', false)
```

Comment:
same comment as the other pr! just use the same prop name,  if you don't see me doing this - tell me so i can fix it hahah